meshroom/ui/qml/main.qml

```diff
@@ -909,8 +909,8 @@ ApplicationWindow {
                     }
                     MaterialToolButton {
                         id: filePollerRefreshStatus
-                        text: _reconstruction.filePollerRefresh == 0 ? MaterialIcons.sync : MaterialIcons.sync_disabled
-                        ToolTip.text: "Set File Refresh Status"
+                        text: (_reconstruction.filePollerRefresh == 0 || _reconstruction.filePollerRefresh == 2) ? MaterialIcons.sync : MaterialIcons.sync_disabled
+                        ToolTip.text: "Set File Watcher Refresh Status"
```

Comment:
From a user perspective, we do not care about the notion of file watcher.

ToolTip.text: "Auto-Refresh Nodes Status For External Changes. \n\n"+
                     "If the status of a node is changed by another instance \n" +
                     "on the same network, such as when computing in render farm."

Reply:
Warning: this string is update somewhere with " (Current: STATUS)" but I do not see where.

Reply:
Done.